source/crud.txt

```diff
@@ -0,0 +1,25 @@
+.. _java-crud-operations:
+
+===============
+CRUD Operations
+===============
+
+.. toctree::
+   :caption: CRUD Operations
+
+   Read </crud/read-operations>
+   Write </crud/write-operations>
+   Query </crud/query-document>
+   Compound Operations </crud/compound-operations>
+
+CRUD (Create, Read, Update, Delete) operations enable you to work with
+data stored in MongoDB.
+
+- :doc:`Read Operations </crud/read-operations>` find and return
+  documents stored in your database.
+- :doc:`Write Operations </crud/write-operations>` insert, modify,
```

Comment:
S: Use :ref: here instead of :doc: - looks like you'd need to ref anchors to the read and write pages